fastlane/Fastfile

```diff
@@ -760,66 +761,56 @@ REPOSITORY_NAME = 'WordPress-Android'
       github_release_prefix: "",
       exclusions: []
     },
-  ]
+  ].freeze
 
-  private_lane :send_strings_for_translation do |options|
-    sh("cd .. && mkdir -p #{update_strings_path} && cp #{main_strings_path} #{update_strings_path} && git add #{update_strings_path}strings.xml")
-    sh('git diff-index --quiet HEAD || git commit -m "Send strings to translation."')
-    sh('git push origin HEAD')
-  end
-
-  private_lane :commit_strings do |options|
-    if options[:auto_commit] then
-       sh("cd .. && git add #{main_strings_path}")
-       sh("git commit -m 'Update strings for translation'")
-       sh('git push origin HEAD')
-    else
-      UI.important("Your #{main_strings_path} has changed.")
-      UI.input('Please, review the changes, commit them and press return to continue.')
+  lane :update_frozen_strings_for_translation do |options|
+    Dir.chdir('..') do
+      FileUtils.mkdir_p(FROZEN_STRINGS_DIR_PATH)
+      FileUtils.cp(MAIN_STRINGS_PATH, FROZEN_STRINGS_DIR_PATH)
     end
```

Comment:
Note: we need the `chdir` because, in the context of commands run directly from the `Fastfile`, the current directory (`pwd`) is `./fastlane`, not the repository root.

(By contrast, in the context of code run from inside a fastlane action (like for `git_commit` below, or any action implemented in our `release-toolkit`), the current directory is the repository root in those other cases).

See [this paragraph in fastlane docs](https://docs.fastlane.tools/advanced/fastlane/#:~:text=Notice%20how%20every,standard%20Ruby%20blocks%3A) for more details.

Reply:
Suggest-Commenting back here what @mokagio suggested [down below](https://github.com/wordpress-mobile/WordPress-Android/pull/15966/files#r808676321) as his suggestion would otherwise conflict with [that other one](https://github.com/wordpress-mobile/WordPress-Android/pull/15966/files#r808711791) he made and that I also wanted to apply.

```suggestion
    # We need to `cd` to the parent directory because, unlike when calling fastlane actions, commands running directly from the `Fastfile`
    # (like `FileUtils` calls here) run relative to the `./fastlane` folder, but the `*_DIR_PATH` we use are relative to the repo root.
    # See: https://docs.fastlane.tools/advanced/fastlane/#directory-behavior
    Dir.chdir('..') do
      FileUtils.mkdir_p(FROZEN_STRINGS_DIR_PATH)
      FileUtils.cp(MAIN_STRINGS_PATH, FROZEN_STRINGS_DIR_PATH)
    end
```